source/content/environment-indicator.md

```diff
@@ -70,111 +84,131 @@ The [Environment Indicator](https://www.drupal.org/project/environment_indicator
+            $config['environment_indicator.indicator']['name'] = 'Dev';
+            $config['environment_indicator.indicator']['bg_color'] = '#d25e0f';
+            $config['environment_indicator.indicator']['fg_color'] = '#ffffff';
             break;
           case 'test':
-            $conf['environment_indicator_overwritten_name'] = 'Test';
-            $conf['environment_indicator_overwritten_color'] = '#c50707';
-            $conf['environment_indicator_overwritten_text_color'] = '#ffffff';
+            $config['environment_indicator.indicator']['name'] = 'Test';
+            $config['environment_indicator.indicator']['bg_color'] = '#c50707';
+            $config['environment_indicator.indicator']['fg_color'] = '#ffffff';
             break;
           case 'live':
-            $conf['environment_indicator_overwritten_name'] = 'Live!';
-            $conf['environment_indicator_overwritten_color'] = '#4C742C';
-            $conf['environment_indicator_overwritten_text_color'] = '#ffffff';
+            $config['environment_indicator.indicator']['name'] = 'Live!';
+            $config['environment_indicator.indicator']['bg_color'] = '#4C742C';
+            $config['environment_indicator.indicator']['fg_color'] = '#ffffff';
             break;
           default:
-            //Multidev catchall
-            $conf['environment_indicator_overwritten_name'] = 'Multidev';
-            $conf['environment_indicator_overwritten_color'] = '#efd01b';
-            $conf['environment_indicator_overwritten_text_color'] = '#000000';
+            // Multidev catchall
+            $config['environment_indicator.indicator']['name'] = 'Multidev';
+            $config['environment_indicator.indicator']['bg_color'] = '#efd01b';
+            $config['environment_indicator.indicator']['fg_color'] = '#000000';
             break;
         }
-    }
-  ```
+      }
+    ```
+
+  </Tab>
+
+  <Tab title="Drupal 7" id="d7tab">
+
+    ```php
+    /*
+    * Environment Indicator module settings.
+    * see: https://pantheon.io/docs/environment-indicator
+    */
+
+    $conf['environment_indicator_overwrite'] = TRUE;
+      $conf['environment_indicator_overwritten_position'] = 'top';
+      $conf['environment_indicator_overwritten_fixed'] = FALSE;
+
+      if (!defined('PANTHEON_ENVIRONMENT')) {
+          $conf['environment_indicator_overwritten_name'] = 'Local';
+          $conf['environment_indicator_overwritten_color'] = '#808080';
+          $conf['environment_indicator_overwritten_text_color'] = '#ffffff';
+      }
+      // Pantheon Env Specific Config
+      if (isset($_ENV['PANTHEON_ENVIRONMENT'])) {
+          switch ($_ENV['PANTHEON_ENVIRONMENT']) {
+            case 'landos': // Localdev or Lando environments
+              $config['environment_indicator.indicator']['name'] = 'Local Dev';
+              $config['environment_indicator.indicator']['bg_color'] = '#808080';
+              $config['environment_indicator.indicator']['fg_color'] = '#ffffff';
```

Comment:
I'm testing out this code on Drupal 7. Needs a few tweaks.

```suggestion
            case 'lando': // Localdev or Lando environments
              $conf['environment_indicator_overwritten_color'] = 'Local Dev';
              $conf['environment_indicator_overwritten_color'] = '#808080';
              $conf['environment_indicator_overwritten_text_color'] = '#ffffff';
              break;
```
First, "landos" is not right. "lando" works for the `case` statement.

This needs a `break;` line at the end or it will set the "dev" options after overriding.

Also, these variable names don't work for Drupal 7. Need to be `$conf['environment_indicator_overwritten_name'] `, `$conf['environment_indicator_overwritten_color']`, and ` $conf['environment_indicator_overwritten_text_color']`.

The `break;` needs to be added on the Drupal 8 part as well.